### PR TITLE
Prepare Openapi code Merge

### PR DESCRIPTION
**Pull request by doriac11**
# Description
- Add missing copyright
- make sure the make check is passing
- make sure the gosec is passing

# ISSUE TYPE

- Bugfix Pull Request

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Unit tests
- [x] Acceptance tests

### CODE REVIEW

**Comment by terraform-providers-svc** [comment]
Can one of the admins verify this patch?

**Comment by doriac11** [comment]
run e2e test

**Comment by doriac11** [comment]
run e2e test

**Comment by doriac11** [comment]
run e2e test

**Review by shekhar-j — APPROVED** [review]

File: powermax/provider/powermax.env [hunk 1]
```diff
@@ -1,6 +1,6 @@
 TF_ACC=1
-POWERMAX_ENDPOINT=https://<UNISPHERE_IP>:8443
-POWERMAX_USERNAME=
-POWERMAX_PASSWORD=
-POWERMAX_SERIAL_NUMBER=
+POWERMAX_ENDPOINT=http://localhost:3000
```
Comment (nikitajoshi1) [inline powermax/provider/powermax.env:2]: @doriac11 Please revert this.

Comment (doriac11) [reply powermax/provider/powermax.env]: done!

**Comment by doriac11** [comment]
run e2e test

**Pull request merged**